### ISSUE TITLE
Get-PartnerCustomer -Domain returns InternalServerError

### ISSUE THREAD

**Issue by boecskoer** [Microsoft/Partner-Center-PowerShell#91]
# Steps to reproduce

1. login with access token
`Connect-PartnerCenter -AccessToken $AccessToken -ApplicationId $AppId -TenantId $TenantId`
2. query customer by domain name
`Get-PartnerCustomer -Domain "xxxxx.onmicrosoft.com"`

`Ge-PartnerCustomer -CustomerId<GUID>` works as expected

## Expected behavior

Should return customer

## Actual behavior

```
PS C:\> Get-PartnerCustomer -Domain "xxxxx.onmicrosoft.com"
Get-PartnerCustomer : InternalServerError 
At line:1 char:1
+ Get-PartnerCustomer -Domain "xxxxx.onmicrosoft.com"
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Get-PartnerCustomer], PartnerPSException
    + FullyQualifiedErrorId : Microsoft.Store.PartnerCenter.PowerShell.Exceptions.PartnerPSException,Microsoft.Store.PartnerCenter.PowerShell.Commands.GetPartnerCustomer
```

## Environment

PS 5.1.16299.820
PartnerCenter 1.5.1902.5

Update:
Same error also occurs when using 1.5.1902.5 in an Azure Automation Runbook

**Comment by isaiahwilliams** [comment]
Hi @boecskoer, 

We are actively investigating this issue. It appears to be caused by an underlying issue with the Partner Center API. If you have not already, can you open a support ticket through Partner Center. I will keep you posted on what we discover.

**Comment by isaiahwilliams** [comment]
Hi @boecskoer, 

I have identified the root cause for this issue. The necessary changes to the module have been made. Right now we are planning to release an update on 3/15/2019 that will include a fix for this issue. Please let us know if you find any other issues.

**Comment by boecskoer** [comment]
Hi @isaiahwilliams,

thank you for the update. As the issue was already found and will be fixed in the upcoming release, do you still require the mentioned support ticket through Partner Center?

Thanks

**Comment by isaiahwilliams** [comment]
Hi Thomas,

You do not have to create a service ticket. Once the release is available you will be able to update and everything will work. Thank you for letting us know this was happening

Isaiah Williams
________________________________

**Comment by isaiahwilliams** [comment]
Hi @boecskoer, 

We just released version 1.5.1903.5 which includes a fix for this issue. Please update and let us know if you have any issues.
